src/sentry/models/integration.py

```diff
@@ -13,19 +13,37 @@
 from sentry.signals import integration_added
 
 
-class PagerDutyServiceProject(Model):
+class PagerDutyService(Model):
     __core__ = False
 
-    project = FlexibleForeignKey("sentry.Project", db_index=False, db_constraint=False)
     organization_integration = FlexibleForeignKey("sentry.OrganizationIntegration")
     integration_key = models.CharField(max_length=255)
     service_id = models.CharField(max_length=255)
     service_name = models.CharField(max_length=255)
+    date_added = models.DateTimeField(default=timezone.now)
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_pagerdutyservice"
+        unique_together = (("service_id", "organization_integration"),)
+
+
+class PagerDutyServiceProject(Model):
+    __core__ = False
+
+    project = FlexibleForeignKey("sentry.Project", db_index=False, db_constraint=False)
+    pagerduty_service = FlexibleForeignKey("sentry.PagerDutyService")
+    date_added = models.DateTimeField(default=timezone.now)
```

Comment:
@wedamija Side note, is there a reason we don't just make a base model class with this field and inherit from that model everywhere?

Reply:
I've thought about that too. We do have a base class currently, but adding that field to it might add it to a bunch of models, depending on how many are missing it. I'm +1 on standardization though.

Reply:
@wedamija Makes sense. Maybe make another new base class that inherits that existing one and just use it for new code where we want it. Eventually, we can merge them into one but better to do that by gradually switching to the new base class.